src/styles/date.scss

```diff
@@ -0,0 +1,9 @@
+.date {
+    white-space: nowrap;
+    padding-right: 10px;
+    font-size: $font-size-small;
+    color: $date-color;
+    &--expired {
+        color: $date-color-expired;
```

Comment:
date-expired-color

Reply:
date color has expired, so $date-color-expired (what is it first, then his condition). I think so